packages/dynamite/dynamite_end_to_end_test/lib/nested_ofs.openapi.json

```diff
@@ -8,7 +8,13 @@
         "schemas": {
             "Base": {
                 "type": "object",
+                "required": [
+                    "attribute"
+                ],
                 "properties": {
+                    "required": [
+                        "attribute"
+                    ],
```

Comment:
I think this is here by accident?

Reply:
I'm quite sure this one is not intended. It is a new property in the schema and not part of the schema itself. Please take a look at the full diff to see what I mean.

Reply:
I understand but I thought I already fixed it. :thinking: